packages/flutter/lib/src/material/input_decorator.dart

```diff
@@ -4081,6 +4099,20 @@ class InputDecorationTheme with Diagnosticable {
   /// If null, defaults to the [ColorScheme.primary].
   final Color? suffixIconColor;
 
+  /// The constraints to use for [InputDecoration.suffixIconConstraints].
+  ///
+  /// This can be used to modify the [BoxConstraints] surrounding
+  /// [InputDecoration.suffixIcon].
+  ///
+  /// This property is particularly useful for getting the decoration's height
+  /// less than 48px. This can be achieved by setting [isDense] to true and
```

Comment:
Super nit: Should you say "the minimum tappable height" instead of 48px, to avoid these comments being wrong if the constant changes? I'm not sure, maybe that's more confusing that just keeping it as 48px.